.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: release
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
let's build it on every tag created instead of filtering it `v*`. Because creating a tag is an explicit call that we would take so adding a filter on the tag convention is redundant.